test/integration/http/faucet.test.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { FaucetClient } from "../../../src/http";
+import { createPolkadotAPI } from "../../../src/factory";
+import { defaultEndpoint } from "../../config";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { AccountData } from "@polkadot/types/interfaces/balances";
+import { assert } from "../../chai";
+import Big from "big.js";
+
+describe.skip("Faucet", function () {
```

Comment:
why deactivate the test?

Reply:
It's "not yet activated" rather than "deactivated". We need the dockerized faucet service in docker-compose first in order to run this test

Reply:
ah, of course. My bad